test/scripts/test_bytecodecompare_prepare_report.py

```diff
@@ -99,6 +99,58 @@ def test_format_report_should_not_print_anything_if_contract_report_list_is_empt
 
         self.assertEqual(report.format_report(), '')
 
+class TestPrepareReport_Statistics(unittest.TestCase):
+    def test_initialization(self):
+        self.assertEqual(Statistics(), Statistics(0, 0, 0, 0, 0))
+
+    def test_aggregate_bytecode_and_metadata_present(self):
+        statistics = Statistics()
+        statistics.aggregate(FileReport(file_name=Path('F'), contract_reports=[ContractReport('C', 'c.sol', 'B', 'M')]))
+        self.assertEqual(statistics, Statistics(1, 1, 0, 0, 0))
```

Comment:
What are these numbers?

Reply:
Ah just saw it on the other file

Reply:
Expected values of the gathered statistics:
``` python
class Statistics:
    file_count: int = 0
    contract_count: int = 0
    error_count: int = 0
    missing_bytecode_count: int = 0
    missing_metadata_count: int = 0
```